docker/kites/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM debian:latest
+MAINTAINER Simone Zaccariello <simone.zaccariello@gmail.com>
```

Comment:
Two issues here:

1) The MAINTAINER instruction is deprecated
2) This is most useful when publishing images on Docker Hub or other registries.

You can remove this line.

Reply:
To set a label corresponding to the MAINTAINER field you could use:
LABEL maintainer="SvenDowideit@home.org.au"

Reply:
Yes. I think we can remove this entirely in this case, because the Dockerfile is going to be maintained by contributors to this whole project.

Reply:
I removed this line, it's not a problem! 👍